tactics/tactics.ml

```diff
@@ -3312,7 +3309,7 @@ let get_recarg_dest (recargdests,tophyp) =
 *)
 
 let induct_discharge with_evars dests avoid' tac (avoid,ra) names =
-  let avoid = Id.Set.union avoid avoid' in
```

Comment:
After this change, the argument `avoid` seems to be unused. Is that correct?
If that is fine, then it would probably be good to remove the `avoid` argument entirely.

Reply:
Never mind, I don't know how I missed that you are indeed unioning together everything relevant.